### PR TITLE
feat(jest): 67% coverage for store/webrtc/getters

### PR DESCRIPTION
**Pull request by drepram**
### What this PR does 📖
- Increase coverage for store/webrtc/getters by adding an extensive unit test

### Which issue(s) this PR fixes 🔨
- Resolve AP-2797

### CODE REVIEW

**Comment by drepram** [comment]
This PR is related to #4979, due to one part of the code using primitive types and the other doesn't (store/audio).

**Review by stavares843 — APPROVED** [review]

**Comment by stavares843** [comment]
🎉

File: store/webrtc/getters.test.ts [hunk 1]
```diff
@@ -0,0 +1,142 @@
+import * as module from './getters'
```
Comment (stavares843) [inline store/webrtc/getters.test.ts:1]: this is failing due to this import

>    Cannot find module './getters' from 'store/webrtc/getters.test.ts'

**Review by stavares843 — CHANGES_REQUESTED** [review]

**Review by Jekrimo — APPROVED** [review]
Approved pending Sara's fix

**Comment by drepram** [comment]
Hi, the CI is failing because Joe removed these files, one of which was store/webrtc/getters:

image

I will be closing this PR 😄

**Comment by stavares843** [comment]
thanks 🔨

**Pull request closed without merging**